Dockerfile

```
@@ -0,0 +1,18 @@
+FROM python:3.9.2-buster
+LABEL name=vaccine-feed-ingest
+
+RUN useradd -m vaccine && mkdir vaccine-feed-ingest && chown vaccine:vaccine vaccine-feed-ingest
+
+COPY ./ /vaccine-feed-ingest/
+RUN rm -rf /vaccine-feed-ingest/.venv/ && mkdir /vaccine-feed-ingest/out && chown vaccine:vaccine /vaccine-feed-ingest/out
+
+USER vaccine
+
+WORKDIR /vaccine-feed-ingest
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+ENV PATH "/home/vaccine/.poetry/bin:$PATH"
+RUN /home/vaccine/.poetry/bin/poetry install --extras lint
```

Since you are installing in a container, you probably don't also need a venv. You can turn off venv with:
```poetry config virtualenvs.create false```

```suggestion
RUN /home/vaccine/.poetry/bin/poetry config virtualenvs.create false && /home/vaccine/.poetry/bin/poetry install --extras lint --no-interaction --no-ansi
```

---

Newest commit removes the .venv